src/Model.php

```diff
@@ -1228,7 +1228,7 @@ public function addCondition($field, $operator = null, $value = null)
         $f = is_string($field) ? $this->getField($field) : ($field instanceof Field ? $field : false);
         if ($f) {
             if ($operator === '=' || func_num_args() == 2) {
-                $v = $operator === '=' ? $value : $operator;
+                $v = ($operator === '=' ? $value : $operator);
```

Comment:
brackets are not needed

Reply:
I know. Added them for readability. I really don't like when there are to many `=` in one line :)

Reply:
Once PSR-12 CS rules will be implemented I think they will be removed :)